app/models/page.rb

```diff
@@ -324,14 +328,16 @@ def export_as_lara_activity(position)
           page_json[:embeddables] << page_element.embeddable.export_as_lara_activity
 
         when "Embeddable::Diy::Sensor"
-          page_json[:show_interactive] = true
-          page_json[:interactives] << page_element.embeddable.export_as_lara_activity
-
         when "Embeddable::Diy::EmbeddedModel"
           page_json[:show_interactive] = true
           page_json[:interactives] << page_element.embeddable.export_as_lara_activity
+          if page_element.embeddable.diy_model.model_type.name == "Digital Microscope"
+            labbook_export[:custom_action_label] = "Upload Image"
```

Comment:
The custom_action_label should be "Take a Snapshot". I know that doesn't make sense, but it is what we had before and Carolyn wants to keep it that.